build-system/global-configs/canary-config.json

```diff
@@ -27,5 +27,6 @@
   "amp-selector": 1,
   "sticky-ad-early-load": 1,
   "amp-auto-ads": 1,
-  "slidescroll-disable-css-snap": 1
+  "slidescroll-disable-css-snap": 1,
+  "a4a-adsense-use-new-format": 1
```

Comment:
Curses!  I wanted to ask you to add a comment here, cross-referencing to your comments in `dep-check-config.js`.  But I don't think there's a way to have comments in JSON.  :-P

More concretely, though, I suggest using a name other than 'new-format' here.  As one Bow reviewer pointed out to me once, 'new' tends to end up super-overloaded and the next thing you know, you've got 'new-new', 'really-new', 'newest', 'even-newer-than-that', etc.  I know that this is intended to be short-term, but the A4A rollout was intended to be short in the first place.  _rolls eyes_